guidelines/terms/22/minimum-bounding-box.html

```diff
@@ -1,6 +1,6 @@
-<dt class="new"><dfn data-lt="bounding boxes">bounding box</dfn></dt>
+<dt class="new"><dfn data-lt="bounding boxes">minimim bounding box</dfn></dt>
 <dd class="new">
 	<p class="change">New</p>	
-   <p>the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie.</p>
+   <p>the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie. For components which wrap onto multiple lines as part of a sentence or block of text (such as hypertext links), the bounding box is based on how the component would appear on a single line.</p>
```

Comment:
i'm still unclear what that second part actually means in practice...how would I imagine a link on a single line? it's a bit cryptic

Reply:
this also makes it confusing when considering targets that *are* fragmented on purpose somehow...would the "pretend it's all together" approach apply to them too?

Reply:
ah, as you were, I see you answered this in the email

> The second sentence about wrapping targets should not apply to target size because, by the definition of target, it is “a region” rather than multiple regions.
>
> It should also not come up due to the inline exception.

Reply:
however, are you re-applying this second sentence here in the PR? or am I getting confused?

Reply:
It is applicable to focus-appearance, and yes, if the link were not wrapped, that's the size.
It is not applicable to target-size, both due to the inline exception, and because a target is defined as a region (not multiple regions).